roxie/ccd/ccdcontext.cpp

```diff
@@ -1298,7 +1299,7 @@ class CRoxieContextBase : implements IRoxieAgentContext, implements ICodeContext
     // interface IRoxieServerContext
     virtual bool collectingDetailedStatistics() const
     {
-        return (workUnit != nullptr);
```

Comment:
I think graphStats might be set up too late for the call from CRoxieServerParallelGraphLoopActivity()
better might be workUnit || statsWu

Reply:
Will change